src/coreclr/jit/gentree.h

```diff
@@ -9748,7 +9760,8 @@ inline bool GenTree::IsMaskZero() const
             assert(op1->OperIsHWIntrinsic());
             id = op1->AsHWIntrinsic()->GetHWIntrinsicId();
         }
-        return ((id >= NI_Sve_CreateFalseMaskByte) && (id <= NI_Sve_CreateFalseMaskUInt64));
+        return ((id == NI_Sve_CreateFalseMaskAll) ||
```

Comment:
This one seems unnecessary, since we should be able to just have a `CNS_MSK` or `CNS_VEC` that is all bits zero, much like we do have all bits set being `true`.